python/lsst/obs/lsstSim/lsstSimMapper.py

```diff
@@ -263,21 +260,19 @@ def _setCcdExposureId(self, propertyList, dataId):
 ###############################################################################
 
     def std_raw(self, item, dataId):
-        exposure = super(LsstSimMapper, self).std_raw(item, dataId)
-        md = exposure.getMetadata()
+        md = item.getMetadata()
         if md.exists("VERSION") and md.getInt("VERSION") < 16952:
-            # Precess crval of WCS from date of observation to J2000
-            epoch = exposure.getInfo().getVisitInfo().getDate().get(dafBase.DateTime.EPOCH)
-            if math.isnan(epoch):
-                raise RuntimeError("Date not found in raw exposure %s" % (dataId,))
-            wcs = exposure.getWcs()
-            origin = wcs.getSkyOrigin()
-            refCoord = afwCoord.Fk5Coord(
-                origin.getLongitude(), origin.getLatitude(), epoch).toIcrs()
-            wcs = makeSkyWcs(crpix=wcs.getPixelOrigin(), crval=refCoord, cdMatrix=wcs.getCdMatrix())
-            exposure.setWcs(wcs)
-
-        return exposure
+            # CRVAL is FK5 at date of observation
```

Comment:
I'm trying to understand the changes you made here.  Before, the code was converting the wcs from an an Fk5 to Icrs.  The new code only changes the metadata.  Is this sufficient?

Reply:
All our use of RA/Dec is now ICRS -- SkyWcs normalizes WCS to ICRS RA, Dec even from FITS-WCS metadata that specifies something else (for instance SDSS images use ICRS Dec, RA and many telescopes use FK5). So there is no need to convert to ICRS.